### ISSUE TITLE
negative samples in VALIDATION set?

### ISSUE THREAD

**Issue by dkashkin** [AlexeyAB/darknet#3162]
@AlexeyAB  this is not a bug just a question. By the way thank you very much for supporting this project! Your help is super appreciated.
My question is - is there any benefit in including the Negative Samples (images with empty annotation TXT files) in the Validation set or should I keep them just in my Training set?

**Comment by AlexeyAB** [comment]
@dkashkin You should keep them just in Training set.

Validation set only participates in the verification of the mAP, but not in the training.

**Comment by dkashkin** [comment]
Thanks @AlexeyAB! This makes sense.

**Comment by willbattel** [comment]
> @dkashkin You should keep them just in Training set.
> 
> Validation set only participates in the verification of the mAP, but not in the training.

@AlexeyAB so even though it won't help any as you've stated- will there be any _penalty_ to having negative samples in the validation set, other than the fact that there are then fewer training images?

**Comment by AlexeyAB** [comment]
@willbattel 

> will there be any penalty to having negative samples in the validation set,

No.

**Comment by ibrahimLearning** [comment]
@AlexeyAB 
Thanks for this issue.
If I use negative sample without any objects with empty label txt-file in training folder, what will happen on the network?
Will it affect the training or detection?
or just ignore it?
Thanks

**Comment by ibrahimLearning** [comment]
@AlexeyAB 
Is there any effect on the training data or detection performance, if I use negative samples in the training folder with empty label-txt files?
That means If I use negative samples in the training folder, the negative samples will be used by network or just ignore it?

Thanks
